client/src/App.tsx

```diff
@@ -4,23 +4,35 @@ import {
     Home,
     Profile
 } from "./pages"
+import SignUpForm, { SignUpInformation } from "./pages/SignUpForm"
+import { useState } from "react"
+
+
 
 function App() {
+    const [signUpInformation, setSignUpInformation] = useState({
+        name: "",
+        email: "",
+        password: "",
+    })
+    
     return (
-        <Router>
-            <Switch>
-                <Route path="/profile">
-                    <Page>
-                        <Profile />
-                    </Page>
-                </Route>
-                <Route path="/">
-                    <Page>
-                        <Home />
-                    </Page>
-                </Route>
-            </Switch>
-        </Router>
+        <SignUpForm information={signUpInformation} onInformationChange={setSignUpInformation}/>
```

Comment:
Could you make a `Route` for the sign up page and put `SignUpForm` in there? I think we should still keep the `Router` in here.

Reply:
Will revert changes to old App.tsx version

Reply:
Done.